src/Config/MosaicConfig.ts

```diff
@@ -1,47 +1,147 @@
-import * as fs from 'fs-extra';
 import * as path from 'path';
```

Comment:
Why `MosaicConfig.ts` file has classes `OriginLibraries`, `OriginChain`, `OriginContracts`, `AuxiliaryContracts`, `ContractAddresses` and `AuxiliaryChain` classes ? 
I think we should have these classes in separate files under `Config` ? What do you think ?

Reply:
As discussed, These are small classes and with no behavior currently. It's fine for now to keep them in one file for now. If the logic expands in the future they can be moved to separate files.